config/prov/prov_test.yaml

```diff
@@ -128,11 +128,14 @@ k8s_cortx_deploy:
   s3_max_start_timeout: 240
   setup_size: "small"
   protocol: "http://{}:{}"
+  https_protocol: "https://{}"
   https_port: 30443
   http_port: 30080
   control_port_https: 31169
   iface: "eth1"
   service_type: "LoadBalancer"
+  lb_count: 1
+  deployment_type: "standard"
```

Comment:
just for my understanding, what other options for deployment type?

Reply:
we have 2 options standard and data-only